tslint-warnings.csv

```diff
@@ -218,7 +218,7 @@ CWE 710 - Coding Standards Violation"
 no-void-expression,Requires expressions of type `void` to appear in statement position.,TSLINT8CA59S,tslint,Non-SDL,Warning,Important,Opportunity for Excellence,See description on the tslint or tslint-microsoft-contrib website,TSLint Procedure,,
 no-with-statement,Do not use with statements. Assign the item to a new variable instead,TSLINT185D93L,tslint,Non-SDL,Warning,Important,Opportunity for Excellence,See description on the tslint or tslint-microsoft-contrib website,TSLint Procedure,"398, 710","CWE 398 - Indicator of Poor Code Quality
 CWE 710 - Coding Standards Violation"
-non-literal-fs-path,Detect calls to fs functions with a non literal filepath,TSLINTJNBF48,tslint,SDL,Error,Critical,Mandatory,See description on the tslint or tslint-microsoft-contrib website,TSLint Procedure,22,"CWE 22 - undefined"
+non-literal-fs-path,Detect calls to fs functions with a non literal filepath,TSLINTJNBF48,tslint,SDL,Error,Critical,Mandatory,See description on the tslint or tslint-microsoft-contrib website,TSLint Procedure,22,"CWE 22 - Improper Limitation of a Pathname to a Restricted Directory ('Path Traversal')"
```

Comment:
Is this related? If not, could you please undo this change? It might have slipped in from another pull request...

Reply:
Ahh, the rest of the code looks good - this is fine 🙂

Reply:
Wow this was fast, I was about to make a commit for updating documentation, but then I saw PR was merged 😄  Thanks @JoshuaKGoldberg